README.rst

```diff
@@ -83,12 +87,30 @@ drgn can be installed using the package manager on some Linux distributions.
   Install the `drgn <https://aur.archlinux.org/packages/drgn/>`_ package from
   the `AUR <https://wiki.archlinux.org/title/Arch_User_Repository>`_.
 
+* Debian
+
+  Install the `python3-drgn <https://packages.debian.org/search?keywords=python3-drgn>`_
+  package. Note: only on `testing` and `unstable` until `bookworm` is released as `stable`.
```

Comment:
What do you think of condensing this down to:

```suggestion
* Debian >= 12 (Bookworm)

```

Which is similar to what we have for Fedora.